src/PhpBrew/Patches/PHP56WithOpenSSL11Patch.php

```diff
@@ -16,8 +16,10 @@ public function desc()
 
     public function match(Buildable $build, Logger $logger)
     {
-        return version_compare($build->getVersion(), '5.6') === 0
-            && version_compare($build->getVersion(), '5.6.31') >= 0 // patch only works for 5.6.31 and up
+        $buildVersion = $build->getVersion();
+
+        return substr($buildVersion, 0, 4) == '5.6.'
```

Comment:
Please use strict comparison for strings unless otherwise is required.

Reply:
Done!